mon-pix/app/components/challenge-item-generic.js

```diff
@@ -35,6 +35,7 @@ const ChallengeItemGeneric = Component.extend({
     if (!this._isUserAwareThatChallengeIsTimed) {
       this.set('hasUserConfirmWarning', false);
       this.set('hasChallengeTimer', this.hasTimerDefined());
```

Comment:
```suggestion
      this.set('hasChallengeTimer', this.hasChallengeTimer());
```

Reply:
@HEYGUL Une fonction `hasChallengeTimer` existe déjà dans ce fichier. J'ai essayé de fusionner les deux fonctions pour ne garder que `hasChallengeTimer`, ce qui donne:
```
hasChallengeTimer: computed('challenge', 'challenge.timer', function() {
    return _.isInteger(this.challenge.timer);
  })
```
Mais le timer ne s'affiche plus, et dans la console j'ai le warning: `#hasChallengeTimer property was just overridden. This removes the computed property and replaces it with a plain value, and has been deprecated`. Je ne suis pas parvenu à aller au bout de l'idée.

Reply:
Sinon, au lieu de `hasChallengeTimer` on pourrait remplacer par `isTimed`

Reply:
On a vu avec @HEYGUL pour merger les deux fonctions `hasTimerDefined` et `hasChallengeTimer` ;)